Latest/Model/AppDataStore.swift

```diff
@@ -230,10 +230,13 @@ class AppDataStore {
 		self.updateCountOfAvailableApps()
 		self.scheduleFilterUpdate()
 	}
+    
+    /// Default identifiers of ignored apps.
+    private static let defaultIgnoredAppIdentifiers = ["com.apple.Safari"]
```

Comment:
I don't quite get this one. Why would you want to explicitly exclude Safari? There are many system apps that will never be updated outside OS Updates. They would all have to go. On the other hand I always thought of adding System Updates, so Safari might be able to be updated though Latest some day.

Since unsupported apps are disabled by default, I think we can leave that in for now..

Reply:
Actually, Safari is the only system app that appears in the list on my side. I like to display the unsupported third-party apps to see which ones I have to check manually.

I was unsure whether to silently filter it or put it in the ignore list by default.

Reply:
I see. However I am still uncomfortable with building such a ignored list right now. Sometimes there are Safari updates on older OSes IIRC, I don't think we should leave these out, especially since you can't unignore Safari at the moment.

Reply:
It can in fact be unignored but I agree with you. I did not think there could be Safari updates decoupled from the OS.